not-supported/borgbackup.sh

```diff
@@ -21,6 +21,7 @@ root_check
 
 # Variables
 LVM_MOUNT="/system"
+ZFS_MOUNT="/ncdata"
```

Comment:
What's this doing?

Reply:
this is the directory where the ZFS snapshot gets mounted

Reply:
I think the folder name is confusing. Would be better to name it `zfs-backup-snapshots`

Reply:
No, This is the place where the newest snapshot gets mounted from where the data is read that gets backed up (I don't backup from the partition itself but from the snapshot ehich reduces the downtime to a very small timeframe.)

Actually, in my opinion is the name `zfs-backup-snapshots` much more confusing than `ncdata` when you open a backup via the backup viewer because you see the parent directory in the backup itself. 
Please trust me that this directory is the best mountpoint for the snapshot. I've thought about it a lot!

Reply:
Or do you mean the variable name?

Reply:
I mean the folder name. Someone not knowing might mix up the `/ncdata` with `/mnt/ncdata` if they explore the file system. It must be very clear that it's not the actual `ncdata` folder IMHO.

Reply:
It is clear because /ncdata is always empty. It will only be filled with data when the snapshot gets mounted there. I mean I could remove it after the backup is done. What do you think about this idea?

Reply:
I hope I've addresses this review with dd32a52